.github/workflows/snyk.yml

```diff
@@ -1,39 +0,0 @@
-name: snyk
-
-"on":
-  push:
-    branches: [main, dev]
-  pull_request:
-    branches: [main, dev]
-  # You can use the merge_group event to trigger your GitHub Actions workflow when
-  # a pull request is added to a merge queue
-  # https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#triggering-merge-group-checks-with-github-actions
-  merge_group:
-
-jobs:
-  security-scan:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v4
-        with:
-          python-version: "3.11.4"
```

Comment:
decided to refactor this, because of this.. now we only have our backend Python version defined in one location for our GitHub Actions workflows, [here](https://github.com/grafana/oncall/blob/dev/.github/actions/setup-python/action.yml#L19) in `.github/actions/setup-python/action.yml`